include/git2/transaction.h

```diff
@@ -8,6 +8,14 @@
 #define INCLUDE_git_transaction_h__
 
 #include "common.h"
+
+/**
+ * @file git2/transaction.h
+ * @brief Git transaction routines.
+ * @defgroup Git transaction routines.
```

Comment:
I was trying to make docurium document those functions. Wasn't sure if the group being defined was a prerequisite, but I'll take them out then.